### DISCUSSION TITLE
Logo 意见建议收集

### DISCUSSION THREAD

**Discussion by linyinfeng** [NixOS-CN/nixos-cn-artwork#2] [General]

**Comment by Easedge** [comment]
和官方一样，加入分形要素，这个建议如何呢

**Comment by linyinfeng** [comment]
@oxalica 在群组的建议：“有点太像 openai”，（logo 的部件）“说是 人 有一点难想象”，“不知道像 ubuntu 那样加个头会不会好点”。

**Comment by ClSlaid** [comment]
$\lambda$ 是精髓，去掉了是不是太可惜

**Comment by OpenTritium** [comment]
新变体中的“人2”很好看，不那么的 openai 了。

**Comment by wineee** [comment]
~~可以试试加点猛男粉或者中国红新配色~~

**Reply by OpenTritium** [reply]
这个建议作为次要的变体图标，或者作为春节图标，因为这些蓝色是 Nix 血统的象征。

**Comment by CyrusYip** [comment]
https://github.com/nixos-cn/nixos-cn-artwork/discussions/3

投票是只投一个，还是可以投多票？

**Reply by linyinfeng** [reply]
随意？多选吧，已经更新介绍。
